docs/source/contributing/developing.adoc

```diff
@@ -211,12 +211,24 @@ For example, _proxy.feature_ or _cmd-version.feature_.
 ==== Running Integration Tests
 
 By default, the tests are being run against the binary created by `make build`, which is *_$GOPATH/bin/minishift_*. 
-To run the tests, use the following command:
+To run the basic test, use the following command:
 
 ----
 $ make integration
 ----
```

Comment:
We should put a note that `make integration` only runs tests which are tagged as basic.

Reply:
Done